src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
You can use: for (const item of actions), because you don't use index of each array item, only value of item